src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -74,42 +82,153 @@
           "formattedRuleMessage": {
             "formatId": "default",
             "arguments": [
-              "n"
+              "p"
```

Comment:
is there an expanded argument value over a single letter that might be more illustrative?